README.md

```diff
@@ -208,7 +213,7 @@ Here's an example of invoking Platform9 Express against a number of hosts withou
 .
 .
 ```
-Here's an example of invoking Platform9 Express a single host groups (host groups are either "pmo" for Openstack and "pmk" for Kubernetes), performing role deployments (based on metadata defined in /opt/pf9-express/inventory/hosts) and registering them automatically to the control plane
+Here's an example of invoking Platform9 Express a single host group (host groups are either "pmo" for OpenStack and "pmk" for Kubernetes), performing role deployments (based on metadata defined in /opt/pf9-express/inventory/hosts) and registering them automatically to the management plane
```

Comment:
Modify the start to be "Here is an example of invoking Platform9 Express against a single host group..."

Add oxford comma after "(based on metadata defined in /opt/pf9-express/inventory/hosts)"

Reply:
Resolved.